docs/UserGuide.md

```diff
@@ -11,117 +12,273 @@
-  The index **must be a positive integer** 1, 2, 3, ...
+Format: `clear -e [-a]`
+
+> An `-a` optional flag will request the TaskBook to clear all events. 
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `clear -e` <br>
+  Clears events that are marked done.
+* `clear -e -a` <br>
+  Clears all events.
+
+  
+#### Clearing of tasks and events
+Clears all completed tasks and events or clears all tasks and events. <br>
 
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+Format: `clear [-a]`
+
+> An `-a` optional flag will request the TaskBook to clear all tasks and events.
+
+Examples: 
+* `clear` <br>
+  Clears all completed tasks and events.
+* `clear -a` <br>
+  Clears all tasks and events.
+  
+  
+#### Exiting the program
+Format : `exit`
 
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
 
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your preious Address Book.
+       the file that contains the data of your previous Task Book.
        
 ## Command Summary
 
-Command | Format  
+ Command | Format  
 -------- | :-------- 
-Add | `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
-Clear | `clear`
-Delete | `delete INDEX`
-Find | `find KEYWORD [MORE_KEYWORDS]`
-List | `list`
-Help | `help`
-Select | `select INDEX`
+[Add Task](#adding-a-task) | `add TASK_NAME [/desc DESCRIPTION] [/by DEADLINE_DATE_TIME]`
+[Add Event](#adding-an-event) | `add EVENT_NAME /from START_DATE_TIME [> END_DATE_TIME] [/desc DESCRIPTION] `
+[List Task or Event](#listing-tasks) | `list -t|-e [-a]`
+[Edit Task](#editing-a-task) | `edit -t INDEX [/name NEW_TASK_NAME] [/desc NEW_TASK_DESCRIPTION] [/by NEW_DEADLINE_DATE_TIME]`
+[Edit Event](#editing-an-event) | `edit -e INDEX [/name NEW_EVENT_NAME] [/desc NEW_EVENT_DESCRIPTION] [/from NEW_START_DATE_TIME > NEW_END_DATE_TIME]`
+[Mark Task](#marking-a-task-as-completed) | `mark INDEX`
+[Delete Task or Event](#deleting-a-task/event) |`delete -t|-e INDEX`
+[Select Task or Event](#selecting-a-task/event) |`select -t|-e INDEX`
+[Save](#changing-the-save-location) | `save FILEPATH`
+[Help](#viewing-help) | `help [COMMAND]`
+[Undo](#undo-modifications) | `undo`
+[Find](#finding-for-events/tasks) | `find -t|-e KEYWORD [MORE_KEYWORDS]`
+[Clear Tasks or Events](#clearing-completed/uncompleted-tasks/events) |`clear -t|-e [-a]`
+[Clear Tasks and Events](#clearing-completed/uncompleted-tasks and events) |`clear [-a]`
+[Exit](#exiting-the-program) | `exit`
```

Comment:
The 'Command summary' section will be used heavily by our testers. Try to make it easy to use as possible. e.g. give examples for each command.